src/components/TodoItem.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+  handleDeleteTodo: (id: number) => void;
+  isLoading: boolean;
+  key: number;
+  onToggle: (todo: Todo) => void;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  handleDeleteTodo,
+  isLoading,
+  onToggle,
+}) => {
+  const [beingRemoved, setBeingRemoved] = useState(false);
+
+  return (
+    <div
+      data-cy="Todo"
+      className={classNames('todo', { completed: todo.completed })}
+    >
+      {/*eslint-disable-next-line jsx-a11y/label-has-associated-control*/}
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={todo.completed}
```

Comment:
Consider using destruction assignment for `todo` object

Reply:
[comment](https://github.com/mate-academy/react_todo-app-with-api/pull/1312/files/5ccac501359e03d96b0cca76be779c719146674b#r1644287079) not fixed